### PR TITLE
[Merged by Bors] - feat(RIngTheory/SimpleRing): matrix ring over a simple ring is simple

### PR DESCRIPTION
**Pull request by jjaassoonn**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/RingTheory/SimpleRing/Matrix.lean [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import Mathlib.LinearAlgebra.Matrix.Ideal
+import Mathlib.RingTheory.SimpleRing.Basic
+
+/-!
+Matrix ring over simple ring is simple
```
Comment (riccardobrasca) [inline Mathlib/RingTheory/SimpleRing/Matrix.lean:10]:
```suggestion
The matrix ring over a simple ring is simple
```

**Review by riccardobrasca — COMMENTED** [review]
Thanks!

bors d+

**Pull request closed without merging**